docs/tutorial.rst

```diff
@@ -0,0 +1,912 @@
+  ...         dec_column="DEJ2000", flux_columns="Si")
+  >>> sm.read_votable_catalog(filename, table_name="VIII_1000_single", id_column="source_id", ra_column="RAJ2000",
+  ...                         dec_column="DEJ2000", flux_columns="Si")
+
+  >>> # Use the `from_file` method to create SkyModel object without initalizing empty object, optionally specify
+  >>> # the file type.
+  >>> # VOTable required parameters: table_name, id_column, ra_column, dec_column, and flux_columns.
+  >>> # Can also use the file type specific `read_votable_catalog` method.
+  >>> sm = SkyModel.from_file(filename, table_name="VIII_1000_single", id_column="source_id", ra_column="RAJ2000",
+  ...                         dec_column="DEJ2000", flux_columns="Si")
+  >>> sm = SkyModel.from_file(filename, filetype="vot", table_name="VIII_1000_single", id_column="source_id",
+  ...                         ra_column="RAJ2000", dec_column="DEJ2000", flux_columns="Si")
+  >>> sm = SkyModel.from_votable_catalog(filename, table_name="VIII_1000_single", id_column="source_id",
+  ...                                    ra_column="RAJ2000", dec_column="DEJ2000", flux_columns="Si")
+
+d) text
+*******
+.. code-block:: python
+
+  >>> import os
+  >>> from pyradiosky import SkyModel
+  >>> from pyradiosky.data import DATA_PATH
+  >>> sm = SkyModel()
+
+  >>> # Use the `read` method, optionally specify the file type. See Attributes section for required columns of text file.
+  >>> # Can also use the file type specific `read_text_catalog` method.
+  >>> filename = os.path.join(DATA_PATH, "pointsource_catalog.txt")
+  >>> sm.read(filename)
+  >>> sm.read(filename, filetype="text")
+  >>> sm.read_text_catalog(filename)
+
+  >>> # Use the `from_file` method to create SkyModel object without initalizing empty object, optionally specify
+  >>> # the file type.
+  >>> sm = SkyModel.from_file(filename)
+  >>> sm = SkyModel.from_file(filename, filetype="text")
+  >>> sm = SkyModel.from_text_catalog(filename)
+
+e) skyh5
+********
+.. code-block:: python
+
+  >>> import os
+  >>> from pyradiosky import SkyModel
+  >>> from pyradiosky.data import DATA_PATH
+  >>> sm = SkyModel()
+
+  >>> # Use the `read` method, optionally specify the file type. See Attributes section for required columns of text file.
+  >>> # Can also use the file type specific `read_text_catalog` method.
+  >>> filename = os.path.join(DATA_PATH, "gsm_icrs.skyh5")
+  >>> sm.read(filename)
+  >>> sm.read(filename, filetype="skyh5")
+  >>> sm.read_skyh5(filename)
+
+  >>> # Use the `from_file` method to create SkyModel object without initalizing empty object, optionally specify
+  >>> # the file type.
+  >>> sm = SkyModel.from_file(filename)
+  >>> sm = SkyModel.from_file(filename, filetype="skyh5")
+  >>> sm = SkyModel.from_skyh5(filename)
+
+SkyModel: Plotting
+------------------
+
+a) FHD (using extended_model_group attribute)
```

Comment:
I wouldn't call this section "FHD". The fact that the object is constructed from an FHD file is incidental, the real point is the extended_model_group. These section heading show up as an outline on RTD, so I'd pick labels that will help users find the sections they're interested in.

In general, all the section headings below here probably don't need to reference the file type, instead they should describe the highlighted functionality.